### DISCUSSION TITLE
Any plans for AnimateDiff?

### DISCUSSION THREAD

**Discussion by Amin456789** [ssube/onnx-web#410] [Ideas]
Hello.

Thank u for making this,

Is there any plans for AnimateDiff too? it will be great if we can have AnimateDiff to make some animations.
onnx is much faster than torch for cpu so lots of us onnx cpu users will be able to use and enjoy it

kind regards

**Comment by ssube** [comment]
I am interested in adding support for AnimateDiff, if it's possible. It will almost certainly need a new pipeline, but I've written a few of those.

The first challenge will be converting the models with the right shapes. I haven't been able to find a good summary of what those should be, the paper says that a fifth dimension is introduced to the UNet (which means you will need a third copy of the UNet, along with the one for ControlNet and the regular one).

I will look into that, but if you have any good resources other than the paper, please send them over. Sorry about the delay, I've been starting a new job recently.

**Reply by Amin456789** [reply]
hey, thank u for ur answer!
sadly i don't have any resources but at here our friend Adam is working on it to bring it to onnx
maybe u guys should collaborate:
https://github.com/saddam213/OnnxStack
